ui/Viewer.js

```diff
@@ -92,6 +92,28 @@ define( [
 		} );
 
 		this.panel.addShowListener( function() {
+			return this.parts.close.on( 'keydown', function( evt ) {
+				var keystroke = evt.data.getKeystroke();
+
+				// Hide the panel on space key press in close button.
+				if ( keystroke == 32 ) {
+					this.blur();
+					this.hide();
+					evt.data.preventDefault();
```

Comment:
This 3 lines of code:

    this.blur();
    this.hide();
    evt.data.preventDefault();

are used in line [88](https://github.com/cksource/ckeditor-plugin-a11ychecker/pull/206/files#diff-f414f99a42e4f003aafb63716d27d219R88), [100](https://github.com/cksource/ckeditor-plugin-a11ychecker/pull/206/files#diff-f414f99a42e4f003aafb63716d27d219R100), [122](https://github.com/cksource/ckeditor-plugin-a11ychecker/pull/206/files#diff-f414f99a42e4f003aafb63716d27d219R122), maybe we could extract it to the separated method, e.g. `hidePanel`/`closePanel` ?